### PR TITLE
Setup hello microverse

### PR DESCRIPTION
**Pull request by mhassa204**
Did the required changes.
1. Linters fixed
2. Setup a new branch as per requirement
3. README.md file modified
https://github.com/mhassa204/Hello-Microverse/pull/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
+Open a pull request.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```
Comment (Shedrack-Sunday) [inline README.md:124]: - [ ] As suggested by the last reviewer, this is a guide and should not be commented on your code. Kindly remove it 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @mhassa204     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there's one adjustment you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Kudus on linter setup ✔️ 
  
# Required Changes  🌵 

- [x] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirements 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```
Comment (franclobo) [inline README.md:30]: - **[OPTIONAL]** Please 🙏🏽 remove or replace the Microverse logo with your own and ensure the image opens successfully for professional documentation.

**Review by franclobo — APPROVED** [review]
Hi @mhassa204 , 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Good UI ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mhassa204** [comment]
Updated

**Pull request closed without merging**